shuffle/cli/src/shared.rs

```diff
@@ -162,65 +162,69 @@ impl NetworkHome {
     }
 
     pub fn archive_old_key(&self, archived_dir: &Path) -> Result<()> {
-        let old_key_path = self.latest_account_key_path.as_path();
+        let old_key_path = self.key_path_for("latest");
         let archived_key_path = archived_dir.join("dev.key");
         fs::copy(old_key_path, archived_key_path)?;
         Ok(())
     }
 
     pub fn archive_old_address(&self, archived_dir: &Path) -> Result<()> {
-        let old_address_path = self.latest_account_address_path.as_path();
+        let old_address_path = self.address_path_for("latest");
         let archived_address_path = archived_dir.join("address");
         fs::copy(old_address_path, archived_address_path)?;
         Ok(())
     }
 
     pub fn generate_paths_if_nonexistent(&self) -> Result<()> {
-        fs::create_dir_all(self.latest_account_path.as_path())?;
-        fs::create_dir_all(self.test_path.as_path())?;
+        fs::create_dir_all(self.accounts_path.join("latest"))?;
+        fs::create_dir_all(self.accounts_path.join("test"))?;
```

Comment:
we use "latest" and "test" in a lot of places. Maybe we can put them in CONST variables and use those variables instead.

Reply:
I'll follow up w this in another PR